tests/integration/api/mutations/shippingRates/createFlatRateFulfillmentRestriction.test.js

```diff
@@ -0,0 +1,92 @@
+import encodeOpaqueId from "@reactioncommerce/api-utils/encodeOpaqueId.js";
+import importAsString from "@reactioncommerce/api-utils/importAsString.js";
+import Factory from "/tests/util/factory.js";
+import TestApp from "/tests/util/TestApp.js";
+
+const CreateFlatRateFulfillmentRestrictionMutation = importAsString("./CreateFlatRateFulfillmentRestrictionMutation.graphql");
+
+jest.setTimeout(300000);
+
+const internalShopId = "123";
+const opaqueShopId = encodeOpaqueId("reaction/shop", internalShopId); // reaction/shop:123
+const shopName = "Test Shop";
+let createFlatRateFulfillmentRestriction;
+let testApp;
+
+const mockFulfillmentRestriction = {
```

Comment:
instead of creating a mock restriction manually, import the `restrictionSchema` in the test [`factory.js`]( https://github.com//reactioncommerce/reaction/blob/05cc419e3623c98111b984c8a9acd5a4818aaae8/tests/util/factory.js) and use  the factory to create the mock restriction. And fields you want to check in the `expect` can be provided to the mock, i.e `shopId`